DfsPlugin/src/org/workcraft/plugins/dfs/DfsToolsProvider.java

```diff
@@ -18,7 +18,7 @@
     public Iterable<GraphEditorTool> getTools() {
         ArrayList<GraphEditorTool> result = new ArrayList<>();
 
-        result.add(new SelectionTool(false));
```

Comment:
Wait, it looks like the bool isn't used, but instead it's replaced by `false` in the constructor. Something looks very wrong.